mta-archive-builder/scripts/mtaBuild.sh

```diff
@@ -0,0 +1,2 @@
+#! /bin/bash
+java -jar ${MTA_JAR_LOCATION} $@
```

Comment:
I would use "$@" instead of $@ to avoid parameter expansion on that level.

Reply:
agreed

Reply:
@radsoulbeard : I just played around with "$@" vs $@ and I was not able to find a difference. Can you provide a small example where the diff is evident?

Reply:
It's a good practice and might make a difference if the path contains characters subject to word splitting (e.g. spaces)

Reply:
@radsoulbeard and me just dived into this issue. We was not able to find a difference. @benhei: somehow academical, but can you provide a small example where this really makes a difference.